docs/PaymentButtons/README.md

```diff
@@ -44,7 +46,7 @@ class MyViewController: ViewController {
         return payPalButton
```

Comment:
❓ I did struggle with adding view constraints here because it requires knowledge of a merchant app's view layout. Should we assume the merchant has an empty screen at this point and center the button vertically?

Reply:
I think the developers know to take care of the constraints based on their layout. I don't think they expect us to provide constraints in our example.

Reply:
I'm cool with leaving them out for now. It did make me think though, maybe a [GitHub gist](https://docs.github.com/en/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists) would be a good tool to use in the future for onboarding merchants.

Reply:
Interesting.

Reply:
We could add constraints to just center the button on a blank page, don't have to do anything wild just give a basic example so merchants know they have to add them.

Reply:
I'll add some constraints in.